### PR TITLE
443 confirmation modal + 342 delete task

### PR DESCRIPTION
**Pull request by ricardovdheijden**
#443 and #342 

- Configures existing confirm modal in project
- adds delete button for tasks
- implements delete functionality

### CODE REVIEW

**Review by Georgi2704 — APPROVED** [review]

File: apps/wfo-ui-surf/pages/_app.tsx [hunk 1]
```diff
@@ -89,27 +90,31 @@ function CustomApp({
...
-                                                >
-                                                    <QueryParamProvider
-                                                        adapter={NextAdapter}
-                                                        options={{
-                                                            removeDefaultsFromUrl:
-                                                                false,
-                                                            enableBatching:
-                                                                true,
-                                                        }}
+                                                <ConfirmationDialogContextWrapper>
```
Comment (acidjunk) [inline apps/wfo-ui-surf/pages/_app.tsx:93]: No Wfo prefix?

File: packages/orchestrator-ui-components/src/hooks/ProcessesHooks/useDeleteProcess.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    const deleteProcess = async (id: string) => {
+        const response = await fetch(`${processesEndpoint}/${id}`, {
+            method: 'DELETE',
+            headers: requestHeaders,
+        });
+
+        if (response.status < 200 || response.status >= 300) {
+            console.error(response.status, response.body);
+            if (response.status === 401 || response.status === 403) {
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/hooks/ProcessesHooks/useDeleteProcess.ts:24]: I wonder if this is smart.

Maybe add a Toast: so the user know what's happening.

Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/hooks/ProcessesHooks/useDeleteProcess.ts:24]: This is part of the auth setup,  we might want to revisit this outside of this PR

File: apps/wfo-ui-surf/pages/_app.tsx (hunk 1 above)
Comment (ricardovdheijden) [reply apps/wfo-ui-surf/pages/_app.tsx:93]: Done

File: packages/orchestrator-ui-components/src/messages/en-US.json [hunk 3]
```diff
@@ -142,6 +151,8 @@
             "retry": "Retry",
             "resume": "Resume",
             "abort": "Abort",
+            "delete": "Delete",
+            "deleteQuestion": "Are you sure you want to delete {workflowName}?",
```
[the remark this answers is not in the document]
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/messages/en-US.json:155]: done

**Pull request merged**